### PR TITLE
[DOC] Accessibility improvements to Using Strimzi guide

### PR DESCRIPTION
**Pull request by laidan6000**
### Type of change

- Documentation

### Description

Guide(s) updated:

- _Using Strimzi_

Make several changes to improve the accessibility of the Strimzi documentation based on common best practices, e.g. Section 508. These changes can help users of screen readers to access the documentation more easily.

Summary of changes:

- Add more descriptive alt-text to some images / graphics
- Add captions to several tables. In Asciidoc, these table captions will render as figures (figure 1, figure 2).
- Split the [Cluster CA Secrets](https://strimzi.io/docs/operators/master/using.html#cluster_ca_secrets) table into five separate tables. Complex tables with merged headers can be difficult for screen readers to interpret. 

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Write tests
- [ ] Make sure all tests pass
- [X] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [X] Reference relevant issue(s) and close them after merging - NO ISSUES FOUND
- [ ] Update CHANGELOG.md
- [ ] Supply screenshots for visual changes, such as Grafana dashboards

### CODE REVIEW

File: documentation/modules/overview/con-overview-components-topic-operator.adoc [hunk 1]
```diff
@@ -10,7 +10,7 @@ The Topic Operator provides a way of managing topics in a Kafka cluster through
 
 .Example architecture for the Topic Operator
 
-image:topic-operator.png[Topic Operator]
+image:topic-operator.png[The Topic Operator manages topics for Kafka and ZooKeeper clusters via KafkaTopic resources]
```
Comment (tomncooper) [inline documentation/modules/overview/con-overview-components-topic-operator.adoc:13]: "manages topics for the Zookeeper cluster"? I am not sure what that means? Topics don't apply to the ZK cluster?

File: documentation/modules/overview/con-overview-components-kafka-bridge-clients.adoc [hunk 2]
```diff
@@ -17,4 +17,4 @@ External clients can access the Kafka Bridge through an OpenShift Route, a loadb
 
 .HTTP internal and external client integration
 
-image:kafka-bridge.png[Kafka Bridge]
+image:kafka-bridge.png[Internal and external HTTP producers and consumers exchange data with the Kafka Bridge]
```
Comment (scholzj) [inline documentation/modules/overview/con-overview-components-kafka-bridge-clients.adoc:20]:
```suggestion
image:kafka-bridge.png[Internal and external HTTP producers and consumers exchange data with the Kafka brokers through the Kafka Bridge]
```

File: documentation/modules/overview/con-overview-components-topic-operator.adoc (hunk 1 above)
Comment (scholzj) [inline documentation/modules/overview/con-overview-components-topic-operator.adoc:13]: I agree with @tomncooper. The Topic Operator connects to Zookeeper, but only to manage topics in Kafka. So IMHO it should be:

```suggestion
image:topic-operator.png[The Topic Operator manages topics for Kafka cluster via KafkaTopic resources]
```

**Review by scholzj — COMMENTED** [review]
LGTM apart from the two comments.

**Review by ppatierno — APPROVED** [review]
LGTM apart from the comments by Jakub and Tom.

**Review by scholzj — APPROVED** [review]

**Comment by scholzj** [comment]
@laidan6000 Can you look at the conflict? Thanks

File: documentation/modules/overview/con-kafka-concepts-components.adoc [hunk 3]
```diff
@@ -14,7 +14,7 @@ Each of the other Kafka components interact with the Kafka cluster to perform sp
 
 .Kafka component interaction
 
-image:overview/kafka-concepts-supporting-components.png[Kafka components]
+image:overview/kafka-concepts-supporting-components.png[Data flows between several Kafka components and the Kafka cluster]
```
Comment (tombentley) [inline documentation/modules/overview/con-kafka-concepts-components.adoc:17]: Can we name the components so it's more descriptive?

**Review by tombentley — APPROVED** [review]

File: documentation/modules/overview/con-kafka-concepts-components.adoc (hunk 3 above)
Comment (laidan6000) [reply documentation/modules/overview/con-kafka-concepts-components.adoc]: That might make the alt-text a bit lengthy. Based on guidelines, I've added a signpost:

"See the component descriptions after this image."

**Review by tomncooper — APPROVED** [review]

**Pull request merged**